js/index.js

```diff
@@ -0,0 +1,20 @@
+let today=new Date();
+// console.log(today)
+let thisYear=today.getFullYear();
+// console.log(thisYear);
+let footer=document.querySelector('footer');
+
+let copyright = document.createElement('p');
+
+copyright.innerHTML = `Asel Karagazieva ${thisYear}`;
+footer.appendChild(copyright);
+
+let skills=['JavaScript', 'HTML'];
+// console.log(skills);
+let skillsSection=document.getElementById('skills');
+let skillsList=document.querySelector('ul');
```

Comment:
I think here they wanted the skills section to be queried instead of the entire document?

Reply:
So something like:
```
skillsSection.getElementsByTagName('ul');
```